README.MD

```diff
@@ -36,53 +34,59 @@ python3 -m pip install .
 ```
 
 
-Examples
---------
+## Examples
 
 Heres a small example of defining a structure using a BlockDef and FieldTypes and using it to create a Block.
 
 ```py
 
 >>> from supyr_struct import *
-
-
+>>>
+>>>
 >>> asdf = BlockDef('some_block',
 ... UInt32('some_int'),
 ... BytesRaw('some_bytes', SIZE=16),
 ... ENDIAN='>')
-
+>>>
+>>>
 >>> test_block = asdf.build()
 >>> test_block.some_int = 1337
->>> test_block.some_bytes = b'heres a cstring\x00'
-
->>> print(test_block)
-... [ Container, entries:2, some_block
-... [ UInt32, size:4, some_int, 1337 ]
-... [ BytesRaw, size:16, some_bytes, <rawdata> ]
-... ]
-
+>>> test_block['some_bytes'] = b'here\'s a cstring\x00'
+>>>
+>>> print('test_block:', test_block)
+test_block: [ Container, entries:2, some_block
+    [ UInt32, size:4, some_int, 1337 ]
+    [ BytesRaw, size:16, some_bytes, <RAWDATA> ]
+    ]
+>>>
+>>>
 >>> test_block.serialize()
-... bytearray(b'\x00\x00\x059heres a cstring\x00')
+BytearrayBuffer(b"\x00\x00\x059here\'s a cstring\x00")
 ```
 
 Supyr allows forcing endianness to be either big, little, or back to normal on a library scale and/or on individual FieldTypes.
 ```py
 >>> field_types.FieldType.force_little()
 >>> test_block.serialize()
-... bytearray(b'9\x05\x00\x00heres a cstring\x00')
+BytearrayBuffer(b"9\x05\x00\x00here\'s a cstring\x00")
+>>>
 >>> field_types.FieldType.force_normal()
 >>> test_block.serialize()
-... bytearray(b'\x00\x00\x059heres a cstring\x00')
->>> field_types.BUInt32.force_little()
+BytearrayBuffer(b"\x00\x00\x059here\'s a cstring\x00")
+>>>
+>>> field_types.FieldType.force_big()
+>>> test_block.serialize()
+BytearrayBuffer(b"\x00\x00\x059here\'s a cstring\x00")
+>>>
+>>> field_types.FieldType.force_little()
 >>> test_block.serialize()
-... bytearray(b'9\x05\x00\x00heres a cstring\x00')
+BytearrayBuffer(b"9\x05\x00\x00here\'s a cstring\x00")
 ```
 
 
 Take a look at the examples module for some ready-to-run example programs that utilize Supyr in different ways.
 
 
-Who do I talk to?
------------------
+## Who do I talk to?
```

Comment:
We might want to change this section to suggest creating issues on github instead, especially since Moses is taking a break.